### DISCUSSION TITLE
Why are all Docker containers created by CapRover being recreated at the same time?

### DISCUSSION THREAD

**Discussion by musagil** [caprover/caprover#2224] [Q&A]
Hello, 
I'm using CapRover to manage my applications, but I've noticed that all containers created by CapRover are being recreated simultaneously at regular intervals. Containers I create manually (outside of CapRover) do not experience this issue.

Here’s what I’ve observed:

- The recreation happens for all CapRover-managed containers at the same time.
- Manually created Docker containers remain unaffected.
- There are no obvious triggers like webhooks, app updates, or health check failures.
- System resources (CPU, memory, disk space) are sufficient.
- No cron jobs or external scripts seem to be interacting with the containers.

Has anyone else experienced this? Could this be related to CapRover's internal configurations or behavior? Any suggestions on how to debug or resolve this would be greatly appreciated!

![image](https://github.com/user-attachments/assets/7ae41c47-11c1-43fd-8d0d-a4be2c9e7840)

**Comment by githubsaturn** [comment]
Caprover uses Docker swarm to manage your containers. 41 minutes ago in your screenshots might indicate a time where the system was restarted.
